cinema/models.py

```diff
@@ -35,12 +38,23 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    filename = (
+        f"{slugify(instance.title)}-{uuid.uuid4()}"
+        + pathlib.Path(filename).suffix
+    )
+    return pathlib.Path("upload/movies/") / pathlib.Path(filename)
```

Comment:
You are converting `filename` 2 times here, you don't need to do it in second one(return statment)